composer.json

```diff
@@ -32,7 +32,7 @@
         "doctrine/inflector": "^1.4 || ^2.0",
         "doctrine/instantiator": "^1.3",
         "doctrine/lexer": "^1.2.3",
-        "doctrine/persistence": "^2.4 || ^3",
+        "doctrine/persistence": "^2.4 || ^3.1",
```

Comment:
Shouldn't this 3.0 version skip at least happen in a minor release instead of a patch one?

Reply:
Sorry, I'll revert this, it's no longer necessary now that the test handles both cases.